src/component/HorizontalFooter/index.jsx

```diff
@@ -1,104 +1,104 @@
-import GenerateLogo from "../../assets/images/landingpage/affiliateorgintros/Generate Logo_Nav.png";
-import InstaIcon from "../../assets/images/socialMediaIcons/Insta.png";
-import LinkedInIcon from "../../assets/images/socialMediaIcons/Linkedin.png";
-import FbIcon from "../../assets/images/socialMediaIcons/Facebook.png";
-import YoutubeIcon from "../../assets/images/socialMediaIcons/Youtube.png";
-import MosaicIcon from "../../assets/images/socialMediaIcons/Mosaic.png";
-import NortheasternIcon from "../../assets/images/socialMediaIcons/Northeaster.png";
-import Sherm from "../../assets/images/socialMediaIcons/Sherm.png";
+// import React, { Component } from "react";
+// import { Container, Row, Col } from "react-bootstrap";
+// import "./style.css";
+// import GenerateLogo from "../../assets/images/landingpage/affiliateorgintros/Generate Logo_Nav.png";
+// import InstaIcon from "../../assets/images/socialMediaIcons/Insta.png";
+// import LinkedInIcon from "../../assets/images/socialMediaIcons/Linkedin.png";
+// import FbIcon from "../../assets/images/socialMediaIcons/Facebook.png";
+// import YoutubeIcon from "../../assets/images/socialMediaIcons/Youtube.png";
+// import MosaicIcon from "../../assets/images/socialMediaIcons/Mosaic.png";
+// import NortheasternIcon from "../../assets/images/socialMediaIcons/Northeaster.png";
+// import Sherm from "../../assets/images/socialMediaIcons/Sherm.png";
 
-export default function HorizontalFooter() {
-  return (
-    <Container fluid className="footer-container">
-      <Row>
-      <div className="rectangleLogo">
-        <div className="logo-background sherm">
-           <a href="https://generatenu.com/">
-            <img src={GenerateLogo} height="50vh" width="50vw" alt="Logo description" />
-          </a>
-        </div>
-      </div>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/about">About </a>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/culture">Culture</a>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/">Teams</a>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/">People</a>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/apply">Apply</a>
-      </Row>
-      <Row className="footer-text">
-        {" "}
-        <a href="/">Projects</a>
-      </Row>
+// export default function HorizontalFooter() {
+//   return (
+//     <Container fluid className="footer-container">
+//       <Row>
+//       <div className="rectangleLogo">
+//         <div className="logo-background sherm">
+//            <a href="https://generatenu.com/">
+//             <img src={GenerateLogo} height="50vh" width="50vw" alt="Logo description" />
+//           </a>
+//         </div>
+//       </div>
+//       </Row>
+//       <Row className="footer-text">
+//         {" "}
+//         <a href="/about">About </a>
```

Comment:
Delete?

Reply:
index.jsx (from what I can tell isn't used, will delete. For some reason there's two files in this folder?